src/lexer.coffee

```diff
@@ -1179,10 +1179,7 @@ CSX_IDENTIFIER = /// ^
 ///
 
 # Fragment: <></>
-CSX_FRAGMENT_IDENTIFIER = /// ^
-  (?![\d<])   # Must not start with number or `<`
-  (.{0}(?=>)) # Ends immediately with '>.
-///
+CSX_FRAGMENT_IDENTIFIER = /// ^ ((?=>)) # Ends immediately with '>`. ///
```

Comment:
You want `` `>` ``, not `` '>` ``. And don't you think the comment should go outside of the regex? It describes the entire regex.

Reply:
Actually, it’s odd that this even compiles. I would think that the latter `///` would be considered part of the comment, leaving this an unclosed regex.

Reply:
Looks like you found a bug! #4811